packages/runtime/src/plugins/usePropagateHeaders.ts

```
@@ -38,60 +38,62 @@ export function usePropagateHeaders<TContext extends Record<string, any>>(
-                    }
+              (headers) =>
+                setOptions({
+                  ...options,
+                  // @ts-expect-error TODO: headers can contain null and undefined values. the types are incorrect
+                  headers: {
+                    ...headers,
+                    ...options.headers,
+                  },
+                }),
+            ),
+          (): OnFetchHookDone | void => {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)